test/e2e/scenarios/multiple_clusters.go

```diff
@@ -0,0 +1,14 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Namespace selector based Network Policies for ingress between clusters, selecting multiple remote clusters", func() {
+	PIt("Should allow communication between pods in selected namespace across multiple clusters", func() {
+		By("creating pod1 with lable1 in cluster1 in namespace1")
+		By("creating pod2 with lable2 in cluster2 in namespace1")
+		By("creating network policy with namespace selector in more than 2 clusters")
```

Comment:
I think we can first verify that before adding the NetworkPolicy the communication fails (as expected) and after adding the NetworkPolicy it succeeds. @mangelajo your thoughts?

Reply:
Assuming that you have added a NetworkPolicy in the project/namespace so that default behavior is overridden.

Reply:
Isn't the default behavior to allow all communications. What we can check is that after adding network policy, non-slected pods can't coomunicate.

Reply:
Like a negative test...

I would have a separate test to verify that NPs are working ok.